### PR TITLE
Added recipe for nearpy

### PR DESCRIPTION
**Pull request by mariusvniekerk**
@jakirkham Addresses #3858

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
There are two pending issues to resolve for future versions of this packages

https://github.com/pixelogik/NearPy/issues/67
https://github.com/pixelogik/NearPy/pull/68

File: recipes/nearpy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+source:
+  # No sdist available
+  # xref: https://github.com/pixelogik/NearPy/issues/67
+  # fn: '{{ name }}-{{ version }}.tar.gz'
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # '{{ hash_type }}': '{{ hash_value }}'
+  # ref: https://github.com/pixelogik/NearPy/issues/67
+  git_url: https://github.com/pixelogik/NearPy
+  git_revision: 1.0.0
```
Comment (isuruf) [inline recipes/nearpy/meta.yaml]: Use  the tarball from github. https://github.com/pixelogik/NearPy/releases

Comment (mariusvniekerk) [reply recipes/nearpy/meta.yaml]: is that stable now? doesn't github change hashes on us every now and then?

**Comment by mariusvniekerk** [comment]
ping @isuruf

File: recipes/nearpy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  # No sdist available
+  # xref: https://github.com/pixelogik/NearPy/issues/67
+  url: https://github.com/pixelogik/{{ name }}/archive/{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch: python
+  number: 0
+  # script: python setup.py install  --single-version-externally-managed --record=record.txt
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
```
Comment (jakirkham) [inline recipes/nearpy/meta.yaml]: Guess the commented version could be dropped in the feedstock.

**Comment by jakirkham** [comment]
Thanks @mariusvniekerk and @isuruf.

**Pull request merged**